src/js/other-websites/inpage_menu.js

```diff
@@ -92,28 +96,68 @@ async function getMasks(options = { fetchCustomMasks: false }) {
   return relayAddresses;
 }
 
+function addUsedOnDomain(domainList, currentDomain) {
+  const usedOnDomains = domainList.split(",");
+
+  // Domain already exists in used_on field. Just return the list!
+  if (usedOnDomains.includes(currentDomain)) {
+    return domainList;
+  }
```

Comment:
nit (non-blocking) micro-optimization?: Is `String.split()` and then Array.includes()` faster than `String.includes()`?

If `String.includes()` is as-fast or faster, we could change this to check `domainList.includes(currentDomain)` and `return` before calling `split(",")`

Reply:
It totally works. Great call out and opportunity for optimization. Revised in d5e6221fe717faa6809d549a9e0b85259f67a063